recipe/meta.yaml

```diff
@@ -36,18 +35,16 @@ requirements:
     - python
     - pip
     - cython
-    - incremental >=21.3.0
-    - setuptools >=35.0.2
-    - wheel >=0.29.0
+    - setuptools 
+    - wheel
+    - incremental 21.3.0
```

Comment:
why specific pinning?

Reply:
Packages in host section should be exact pinned.

Reply:
Moreover, it is good practice to pin to oldest possible, in this case 21.3.0 for incremental package. In run section, can use range with the lower limit ideally to be the version specified in the host section.